tools/sysadmin-vm-prep.sh

```
@@ -0,0 +1,138 @@
+    -u "dsom-admin" \
+    --private-key ~/.ssh/id_dsom_ed25519 \
+    --tags fabric \
+    -e "target_host=$TARGET_HOST" \
+    -e "dsom_root=$DSOM_ROOT" \
+    -e "ansible_ssh_extra_args='-o StrictHostKeyChecking=no -o PreferredAuthentications=publickey,password'" \
+    -v
+
+if [ $? -eq 0 ]; then
+    echo -e "\n${GREEN}[SUCCESS] Node $TARGET_HOST is now compliant.${NC}"
+    echo -e "${CYAN}Verification Dashboard (as dsom-admin):${NC}"
+    ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o BatchMode=yes dsom-admin@$TARGET_HOST "printf \"  - OS: %s\n  - User: %s\n  - Root: $DSOM_ROOT\n  - NTP (Chrony): %s\n  - Lynis Score: %s\n\" \"\$(cat /etc/os-release | grep PRETTY_NAME | cut -d= -f2 | tr -d \\\")\" \"\$(id dsom-admin)\" \"\$(chronyc tracking | grep 'System time' | xargs || echo 'NOT_SYNCED')\" \"\$(grep '^lynis_score=' /etc/um-elastic-soc.conf | cut -d= -f2 | grep . || echo 'N/A')\"" 2>/dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)